influencetx/core/tests/test_utils.py

```diff
@@ -1,6 +1,46 @@
+from contextlib import contextmanager
 from unittest import mock
 
-from influencetx.core import constants, utils
+import pytest
+
+from influencetx.core import constants, testing, utils
+
+
+class TestChamberEnum():
+
+    def test_null_input(self):
+        with pytest.raises(ValueError):
+            utils.chamber_enum(None)
+
+    def test_invalid_input(self):
+        with assert_log_warning():
+            assert utils.chamber_enum('not valid') is None
+
+    def test_lower_chamber(self):
+        assert utils.chamber_enum('lower') == constants.Chamber.LOWER
+
+    def test_upper_chamber(self):
+        assert utils.chamber_enum('upper') == constants.Chamber.UPPER
+
+    def test_mixed_case_lower_chamber(self):
+        assert utils.chamber_enum('LoWeR') == constants.Chamber.LOWER
+
+
+class TestChamberLabel():
+
+    def test_null_input(self):
+        with pytest.raises(ValueError):
+            utils.chamber_label(None)
+
+    def test_invalid_input(self):
+        with assert_log_warning():
+            assert utils.chamber_label('not valid') == ''
+
+    def test_lower_chamber_label(self):
+        utils.chamber_label('lower') == constants.CHAMBER_LABELS[constants.Chamber.LOWER]
```

Comment:
The "test_lower_chamber_label" function does not perform an assert like the other tests, nor does "test_higher_chamber_label".  Was there a reason for not needing the assert?

Reply:
Good catch! Yes, there are multiple tests in this file that were missing an assert. Most likely I forgot it once and just copied pasted multiple times. Fixed.